### DISCUSSION TITLE
Explorer tree file as the go to file ui

### DISCUSSION THREAD

**Discussion by wcastand** [microsoft/vscode-discussions#2409] [Extension Development QnA] [answered]
hi, i was wondering if it was possible to have the tree structure of the explorer for the workspace to open as the UI for the "Go to file" cmd

the idea would be i don't need the explorer anymore and when i hit `cmd+p` i see the file tree with the search bar to filter and find the matching file like the current `go to file` but with the file structure of the workspace, letting me create files etc

**Comment by isidorn** [comment, marked as answer]
You can not currently configure this from an extension.
